src/sentry/models/release.py

```diff
@@ -197,8 +197,6 @@ class Release(Model):
         ),
     )
 
-    # DEPRECATED
-    project_id = BoundedBigIntegerField(null=True)
```

Comment:
Is this to be done in the next PR?
It's unclear from the docs.

Reply:
no -- the actual removal from the database is done in the next PR, this only handles removing references to the column in the code. This is the step from the docs:

>Remove the column from the model, but in the migration make sure we only mark the state as removed.

Reply:
You have to update the django models alongside the state change. This is important because it removes the column from django's field list and effectively doesn't exist to django anymore.